tests/fixtures/issues/issue-525.php

```diff
@@ -1061,59 +1061,6 @@
         'lite' => false,
         'standard' => true,
     ],
-    'issue-525-U' => [
-        'ua' => 'Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10_5_6; en-US) AppleWebKit/530.5 (KHTML, like Gecko) Chrome/ Safari/530.5',
```

Comment:
No chrome version, this one would still be caught by the generic pattern, but I didn't think it should necessarily be included in the test fixtures.